wiki/Manual.md

```diff
@@ -3474,7 +3474,7 @@ We present a unified structure for the positions returned by exchanges.
    'hedged': false,             // boolean, whether or not the position is hedged, i.e. if trading in the opposite direction will close this position or make a new one
    'side': 'long',              // string, long or short
    'contracts': 5,              // float, number of contracts bought, aka the amount or size of the position
-   'contractSize': 100,         // float, the size of one contract in quote units
+   'contractSize': '100',       // string, the size of one contract in quote units
```

Comment:
I don't think this change is correct.

The documentation section is about the "position structure" -> the position return value - which seems to be a number:
https://github.com/ccxt/ccxt/blob/master/js/ftx.js#L1962
https://github.com/ccxt/ccxt/blob/master/js/gateio.js#L2954

Reply:
@xmatthias you're right